src/main/java/io/mbab/sda/groupproject/menu/action/CreateSongAction.java

```diff
@@ -31,7 +31,7 @@ public void execute() {
     System.out.println("Podaj nazwisko autora:");
     input = scanner.nextLine();
     if (pressedZero(input)) return;
-    builder = builder.songAutor(input);
+    builder = builder.songAuthor(input);
```

Comment:
jak masz builder to nie musisz nadpisywac zzmiennej, wystarczy 
builder.songAuthor(input);
bez przypisania builder =